instruction_docs/ACTIVITY_README.md

```diff
@@ -4,35 +4,29 @@
 
 We helped fix Dahlia's code in the Debugging Continued lesson's Problem Set, so we should have an idea of where the bugs are in this starter code and how to fix them. This puts us in a great position to try out debugging with ChatGPT using known issues, allowing us to focus on strategies for prompting rather than finding the bugs.
 
-As a reminder, `main.py` holds the driving code while `part_one.py` and `part_two.py` contain the functions for working with the genetic sequences as strings. `sequence_tests.py` holds tests for both Part 1 and Part 2 of the problem set.
+As a reminder, `main.py` holds the driver code while `source/part_one.py` and `source/part_two.py` contain the functions for working with the genetic sequences as strings. The `tests` folder holds files with tests for both Part 1 and Part 2 of the problem set.
 
 ## Part 1
 
 ### Before Getting Started:
 
-We want to run Dahlia's code and see the same issues, in the same order, as we saw when debugging the problem set. As a group choose someone's existing local repository to use. 
-
-**You may need to revert some changes:**
-1. Any code changes made to fix the bugs
-2. In `main.py` at the top of the file:
-   - comment out the part 2 import statement `from part_two import driver`
-   - uncomment the part 1 import line `# from part_one import driver`
-3. In `sequence_tests.py` comment line 17 `part_two_tests()`
+We want to run Dahlia's code and see the same issues, in the same order, as we saw when debugging the problem set. This means that we want to make a fresh clone from the Ada-Activities repo, rather than using someone's existing repo with fixes in place. Each team member should clone down a new copy of the code from this repo. 
+- Since everyone should already have a folder using the repo name from working on the problem set, you may need to choose a different folder or create a new folder to clone the code into.
```

Comment:
Maybe starting to get a bit into the weeds, but maybe something to remind folks why they need to clone to a different location. If we hint at the error they're like to see, that could let folks understand what's going on and unblock themselves more quickly.

```suggestion
- Since everyone should already have a folder using the repo name from working on the problem set, you may need to choose a different folder or create a new folder to clone the code into, otherwise Git will report that you already have a directory for this repo and that it isn't empty.
```